### ISSUE TITLE
Docker: tree view not working

### ISSUE THREAD

**Issue by megamaced** [LDAPAccountManager/lam#150]
Hi, I am using the :latest tag, new as of today

I have set the docker variable "LDAP_BASE_DN=mydomain.co.uk"

I have checked treesuffix is getting set correctly from start.sh:
```

root@e749f6a19dc1:/var/lib/ldap-account-manager/config# grep tree lam.conf 
# suffix of tree view
treesuffix: DC=mydomain,DC=co,DC=uk
```

However when I click on Tree View I get the error:

`Please configure the tree suffix in your LAM server profile settings.`

**Comment by whlsxl** [comment]
Same error, back to 7.5, it worked

**Comment by gruberroland** [comment]
Will be fixed in 7.7 (this month) as the sample files also need to be changed.
